frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -45,18 +46,23 @@ export const UnsubscribeSurveyModal = ({
                         type={textAreaNotEmpty ? 'primary' : 'secondary'}
                         disabledReason={includesPipelinesAddon && unsubscribeDisabledReason}
                         onClick={() => {
-                            textAreaNotEmpty
-                                ? reportSurveySent(surveyID, surveyResponse)
-                                : reportSurveyDismissed(surveyID)
                             deactivateProduct(product.type)
```

Comment:
Moved these items into a listener in billingProductLogic for deactivateProductSuccess